### PR TITLE
Notification sound update

### PR DESCRIPTION
**Pull request by Sudrien**
Issue #350 is goals & notes. It switches one fallback dependency (flash) for another (bgsound).

### CODE REVIEW

File: src/view/pane/chat.js [hunk 1]
```diff
@@ -293,9 +293,18 @@ Candy.View.Pane = (function(self, $) {
           e.stopPropagation();
         });
         $('#chat-autoscroll-control').click(self.Chat.Toolbar.onAutoscrollControlClick);
-
-        var a = document.createElement('audio');
-        self.Chat.Toolbar._supportsNativeAudio = !!(a.canPlayType && a.canPlayType('audio/mpeg;').replace(/no/, ''));
+		if( !!document.createElement('audio').canPlayType ) {
+			var a = document.createElement('audio');
+			if( !!(a.canPlayType('audio/mpeg;').replace(/no/, '')) ) {
```
Comment (tylermauthe) [inline src/view/pane/chat.js]: This block of if/else tests needs to be wrapped in a try catch for IE9. (See #357 & #356)

**Comment by tylermauthe** [comment]
Confirmed to work with IE9, closing #357 as this PR supersedes it.

File: src/view/pane/message.js [hunk 2]
```diff
@@ -216,7 +216,7 @@ Candy.View.Pane = (function(self, $) {
         // Notify the user about a new private message
         if(Candy.View.getCurrent().roomJid !== roomJid || !self.Window.hasFocus()) {
           self.Chat.increaseUnreadMessages(roomJid);
-          if(Candy.View.Pane.Chat.rooms[roomJid].type === 'chat' && !self.Window.hasFocus()) {
+          if((Candy.View.Pane.Chat.rooms[roomJid].type === 'chat' || Candy.View.getOptions().updateWindowOnAllMessages === true) && !self.Window.hasFocus()) {
```
Comment (mweibel) [inline src/view/pane/message.js]: What does this have to do with the notification sound update?
Will remove this file before merging (it's creating conflicts anyway)

**Comment by Sudrien** [comment]
@mweibel That was supposed to be an extension of the updateWindowOnAllMessages in #349 - yeah, should have been a separate PR
